examples/README.md

```diff
@@ -439,6 +439,16 @@ For this example the following is needed in the selected region:
 
 ### [slurm-gcp-v5-hpc-centos7.yaml] ![community-badge]
 
+> **Warning**: The variables `enable_reconfigure`,
+> `enable_cleanup_compute`, and `enable_cleanup_subscriptions`, if set to
+> `true`, require additional dependencies **to be installed on the system running
+> `terraform apply`**.
```

Comment:
Same for all examples, I wish MD woulda allow you to include the contents from another file.